metricbeat/beater/module.go

```diff
@@ -0,0 +1,204 @@
+package beater
```

Comment:
Same comment as for "event". Having it in its own namespace could make it easier to use as library.